ovos_PHAL_plugin_alsa/__init__.py

```diff
@@ -21,6 +21,11 @@ def __init__(self, bus=None, config=None):
         self.bus.on("mycroft.volume.unmute", self.handle_unmute_request)
         self.bus.on("mycroft.volume.mute.toggle", self.handle_mute_toggle_request)
 
+        # A silent method to get the volume without invoking the shell osd
+        # Needed as gui will always refresh and request it
+        # When sliding panel opens to refresh volume value data
+        self.bus.on("mycroft.volume.get.silent", self.handle_get_silent)
```

Comment:
since the handler user .response internally you can delete `self.handle_get_silent` and use the regular method, the message sent will be different like you need it to